src/data/finsupp/basic.lean

```diff
@@ -861,6 +874,16 @@ lemma map_range_add [add_zero_class N]
   map_range f hf (v₁ + v₂) = map_range f hf v₁ + map_range f hf v₂ :=
 ext $ λ a, by simp only [hf', add_apply, map_range_apply]
 
+@[simp] lemma emb_domain_add (f : α ↪ β) (v w : α →₀ M) :
```

Comment:
Please turn this into a def of `emb_domain.add_monoid_hom` or something like that. So that it can be used easily to prove preservation of `sub, neg, sum` etc.

Reply:
Done.